go/autoscaler/autoscaler.go

```diff
@@ -222,6 +225,30 @@ nextJob:
 	return js
 }
 
+func searchAssignableNodeByCPU(r *ClusterResource, j job) (assignable bool) {
+	assignable = false
+	for _, idle := range r.NodesCPUIdleMilli {
+		if j.TrainerCPURequestMilli() <= idle {
+			assignable = true
```

Comment:
In go, if return value is declared with a var name, `return` will return the var name directly, this is useful when we need to declare the return value at the begining of the function, like `var ret bool`.

In your case, just define return type `bool` and `return false` or `return true` will do the job.

Reply:
Thanks, got it. Done.